cmd/kubectl-gadget/profile/cpu.go

```diff
@@ -112,8 +153,83 @@ func runProfileCPU(cmd *cobra.Command, args []string) error {
 	}
 
 	displayResultsCallback := func(traces []gadgetv1alpha1.Trace) error {
+		// print header
+		switch params.OutputMode {
+		case utils.OutputModeCustomColumns:
+			fmt.Println(getCustomProfileColsHeader(params.CustomColumns))
+		case utils.OutputModeColumns:
+			fmt.Printf("%-16s %-16s %-16s %-16s %-16s %-6s %-6s\n",
+				"NODE", "NAMESPACE", "POD", "CONTAINER", "COMM", "PID", "COUNT")
+		}
+
 		for _, trace := range traces {
-			fmt.Printf("%v\n", trace.Status.Output)
+			var reports []types.Report
+			if err := json.Unmarshal([]byte(trace.Status.Output), &reports); err != nil {
+				return utils.WrapInErrUnmarshalOutput(err, trace.Status.Output)
+			}
+
+			for _, report := range reports {
+				switch params.OutputMode {
+				case utils.OutputModeColumns:
+					var sb strings.Builder
+
+					fmt.Fprintf(&sb, "%-16s %-16s %-16s %-16s %-16s %-7d %-6d\n",
+						report.Node, report.Namespace, report.Pod, report.Container,
+						report.Comm, report.Pid, report.Count)
+
+					if profileUser {
```

Comment:
I know it was there before but the name of this variable is confusing, it should be something like `profileOnlyUser`, otherwise it gives the impression that when not flag is passed then nothing should be printed.